lib/index.js

```diff
@@ -1 +1,3 @@
 require ('./css/styles');
+
+const firebase = require('./firebase');
```

Comment:
I think it's fine to leave for now, we can remove it later if/when we can confirm it isn't needed.